opsdroid/core.py

```diff
@@ -431,3 +450,23 @@ def start_databases(self, databases):
                                        message)))
 
         return tasks
+
+    async def send(self, event):
+        """Send an event.
+
+        If ``event.connector`` is not set this method will use
+        `OpsDroid.default_connector`. If ``event.connector`` is a string, it
+        will be resolved to the name of the connectors configured in this
+        instance.
+
+        Args:
+            event (opsdroid.events.Event): The event to send.
+
+        """
+        if isinstance(event.connector, str):
+            event.connector = self._connector_names[event.connector]
```

Comment:
Overwriting `event.connector` ?

Reply:
yes, it converts it to an actual object rather than the string representation thereof.